istio/tests/cases/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m pip install pipenv
```

Comment:
don't install pipenv. let user install it. make as pre-requsitie

Reply:
ok